tests/Integration/Behaviour/Features/Context/EmployeeFeatureContext.php

```diff
@@ -41,6 +41,6 @@ public function logsInBackOffice($employeeEmail)
     {
         $employeeId = Employee::employeeExists($employeeEmail);
```

Comment:
Use getByEmail instead.
It returns a wrong id otherwise. This test is completely broken

Reply:
Done.